packages/accordion/src/vaadin-accordion.js

```diff
@@ -8,7 +8,6 @@ import { css, html, LitElement } from 'lit';
 import { defineCustomElement } from '@vaadin/component-base/src/define.js';
 import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
 import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
-import { LumoInjectionMixin } from '@vaadin/vaadin-themable-mixin/lumo-injection-mixin.js';
```

Comment:
I think this can be removed in a separate PR - originally mixin was needed in `findRoot` but we changed it to rely on `static get version()` instead.

Reply:
If I don't remove it, I'll have to add `{ includeBaseStyles: true }`, so a change is required either way. Removing it altogether seemed simpler.